config.js

```diff
@@ -71,11 +71,24 @@ const readReplicas = configUtil.getReadReplicas(pe, replicaConfigLabel);
 const DEFAULT_JOB_QUEUE_TTL_SECONDS_ASYNC = 3600;
 const DEFAULT_JOB_QUEUE_TTL_SECONDS_SYNC = 25;
 
-// default set to 30 minutes
-const DEFAULT_JOB_REMOVAL_INTERVAL_SECONDS = 1800;
+const DEFAULT_JOB_REMOVAL_INTERVAL_MINUTES = 30;
+const JOB_REMOVAL_INTERVAL = 60 * 1000 * (pe.KUE_JOBS_REMOVAL_INTERVAL_MINUTES ||
+  DEFAULT_JOB_REMOVAL_INTERVAL_MINUTES);
 
-const JOB_REMOVAL_INTERVAL_SECONDS = pe.KUE_JOBS_REMOVAL_INTERVAL ||
-  DEFAULT_JOB_REMOVAL_INTERVAL_SECONDS;
+const DEFAULT_JOB_REMOVAL_DELAY_MINUTES = 30;
+const JOB_REMOVAL_DELAY = 60 * 1000 * (pe.KUE_JOBS_REMOVAL_DELAY_MINUTES ||
+  DEFAULT_JOB_REMOVAL_DELAY_MINUTES);
+
+const DEFAULT_JOB_REMOVAL_WINDOW = 1000;
+const JOB_REMOVAL_WINDOW = pe.KUE_JOBS_REMOVAL_WINDOW ||
+  DEFAULT_JOB_REMOVAL_WINDOW;
+
+// This must be set to several times the value of JOB_REMOVAL_INTERVAL or
+// completed jobs may be overwritten sooner that JOB_REMOVAL_DELAY
```

Comment:
What about making the delay config var a multiplier of the interval minutes rather than a number of minutes itself? And if user provides any multiplier less than 1.5 just set it to 1.5, or something like that?